tools/postgres-init-dev-db

```diff
@@ -24,7 +24,7 @@ if [[ $# == 0 ]]; then
     USERNAME=zulip
     PASSWORD=$("$(dirname "$0")/../bin/get-django-setting" LOCAL_DATABASE_PASSWORD)
     DBNAME=zulip
-    SEARCH_PATH="$USERNAME",public
+    SEARCH_PATH="$USERNAME",public,pgroonga,pg_catalog
```

Comment:
Ah, should we use `ADDITIONAL_PACKAGES` environment variable introduced at https://github.com/zulip/zulip/commit/68c6d514e8a18e8c5d8af60b9fa425cb7e87c9d1 ?